openaq_api/openaq_api/main.py

```diff
@@ -51,7 +51,6 @@
     locations,
     manufacturers,
     measurements,
-    owners,
```

Comment:
Can we add this back in, i think this got inadvertently removed from the branching issue.

Reply:
No problem. All set. I am not sure why its showing a spacing issue still, I am not seeing that on my end but let me know if it becomes a problem.